userguide/aws/how-to-set-up-a-cloud-formation-system.adoc

```diff
@@ -5,11 +5,11 @@
 
 This document describes the procedures for setting up Kill Bill under AWS using the CloudFormation option. This is one of two recommended alternatives for production use. The multi-tier option requires more setup than CloudFormation, but provides more control over the deployment. The procedures in this document are based on the options recommended by Kill Bill.
 
-Deployment via CloudFormation leverages the capabilites of the AWS infrastructure to provide a robust *production ready* deployment with a single click. The entire CloudFormation configuration, or stack, is defined by a *CloudFormation Template*.
+Deployment via CloudFormation leverages the capabilities of the AWS infrastructure to provide a robust *production ready* deployment with a single click. The entire CloudFormation configuration, or stack, is defined by a *CloudFormation Template*.
 
 The features of the CloudFormation system include:
 
-* Both Kill Bill and Kaui instances can be scaled up or down using AWS autoscaling groups
+* Kill Bill and Kaui instance can be scaled up or down using AWS autoscaling groups
```

Comment:
Consider rephrasing as 
```suggestion
* The instance having Kill Bill/Kaui installed can be scaled up or down using AWS autoscaling groups
```
or something similar to make it clear that KB/Kaui are on the same instance.